jina/types/arrays/neural_ops.py

```diff
@@ -267,13 +267,23 @@ def embeddings(self) -> np.ndarray:
 
         :return: embeddings stacked per row as `np.ndarray`.
         """
-
         x_mat = b''.join(d.proto.embedding.dense.buffer for d in self)
 
         return np.frombuffer(x_mat, dtype=self[0].proto.embedding.dense.dtype).reshape(
             (len(self), self[0].proto.embedding.dense.shape[0])
         )
 
+    @embeddings.setter
+    def embeddings(self, emb: np.ndarray):
```

Comment:
not sure it belongs here, but good for now

Reply:
it it is the only mixin that we have that covers DA and DAM that is why it's here..